pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -396,6 +396,7 @@ func getTPMapMatchingExistingAntiAffinity(pod *v1.Pod, nodeInfoMap map[string]*s
 		node := nodeInfo.Node()
 		if node == nil {
 			catchError(fmt.Errorf("node not found"))
```

Comment:
Would you mind updating to:

```suggestion
			catchError("node %q not found", node.Name)
```